src/FastRouteRouterFactory.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
+ * @copyright Copyright (c) 2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Zend\Expressive\Router;
+
+use Interop\Container\ContainerInterface;
+
+/**
+ * Create and return an instance of FastRouteRouter.
+ *
+ * Configuration should look like the following:
+ *
+ * <code>
+ * 'router' => [
+ *     'fastroute' => [
+ *         'cache_enabled' => true, // true|false
+ *         'cache_file'   => '(/absolute/)path/to/cache/file', // optional
+ *     ],
+ * ]
+ * </code>
+ */
+class FastRouteRouterFactory
```

Comment:
@Ocramius 
The factory is needed to inject configuration parameters from the container config to the router constructor (btw initially the constructor had 2 extra params $cacheFile and $chacheEnabled, no need for special config keys inside the class, but afterwards i decided to use one $config as it was done in other components)  Since the default properties are set not to use caching, the factory would be needed only for those who want to use caching, but i don't think always using the factory could have any impact on performance as the router is created only once.

in my view  
- the expressive setup would always have the factory taking care of the router creation.
- the factory would examine the configuration and pass a $config array to the router constructor.
- the router would be using or not using caching based on the configuration params.